.gitmodules

```diff
@@ -1,7 +1,3 @@
-[submodule "xen"]
```

Comment:
Why did you remove this submodule?

Reply:
I updated my xen version and made some changes., And forgot to review while making a update on PR..